Twig/RollbarExtension.php

```diff
@@ -128,8 +128,24 @@ function isFromAllowedHosts(filename) {
         return false;
     }
 
+    function isStringError(payload) {
+        try {
+            if (payload.data.body.message.body !== undefined) {
```

Comment:
From docs (https://rollbar.com/docs/api/items_post/):
```
...
"body": {
      ...
      // Option 3: "message"
      // Only one of "trace", "trace_chain", "message", or "crash_report" should be present.
      // Presence of a "message" key means that this payload is a log message.
      "message": {
      ...
```

So I guess that check for `payload.data.body.message` should be enough.